File: src/C#/CreationalPatterns/AbstractFactory/Furniture/ConcreteFactories/ScandinavianFurnitureFactory.cs
using Furniture.Interfaces;
using Furniture.Products;
using System;
using System.Collections.Generic;
using System.Text;

namespace Furniture.ConcreteFactories
{
    public class ScandinavianFurnitureFactory : IFurnitureFactory
    {
        public IFurniture CreateCabinet()
        {
            return new ScandinavianCabinet();
        }

        public IFurniture CreateChair()
        {
            return new ScandinavianChair();
        }

        public IFurniture CreateDiningTable()
        {
            return new ScandinavianDiningTable();
        }
    }
}

File: src/Java/CreationalPatterns/AbstractFactory/Furniture/src/ConcreteFactories/ScandinavianFurnitureFactory.java
package ConcreteFactories;

import Interfaces.IFurniture;
import Interfaces.IFurnitureFactory;
import Products.ScandinavianCabinet;
import Products.ScandinavianChair;
import Products.ScandinavianDiningTable;

public class ScandinavianFurnitureFactory implements IFurnitureFactory {
    @Override
    public IFurniture createCabinet() {
        return new ScandinavianCabinet();
    }

    @Override
    public IFurniture createChair() {
        return new ScandinavianChair();
    }

    @Override
    public IFurniture createDiningTable() {
        return new ScandinavianDiningTable();
    }
}
